torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <torch/nn/options/activation.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn{
+namespace functional {
+
+inline Tensor elu(const Tensor& input, const ELUOptions& options) {
+  if (options.inplace()) {
+    return torch::elu_(const_cast<Tensor&>(input), options.alpha());
```

Comment:
hmm I think we can consider changing `input`'s type from `const Tensor&` to `Tensor&`, since `elu` can be an in-place operation. After that change, we won't need `const_cast` here.